fe/fe-core/src/main/java/org/apache/doris/load/GroupCommitManager.java

```diff
@@ -0,0 +1,171 @@
+// Licensed to the Apache Software Foundation (ASF) under one
+// or more contributor license agreements.  See the NOTICE file
+// distributed with this work for additional information
+// regarding copyright ownership.  The ASF licenses this file
+// to you under the Apache License, Version 2.0 (the
+// "License"); you may not use this file except in compliance
+// with the License.  You may obtain a copy of the License at
+//
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+package org.apache.doris.load;
+
+
+import org.apache.doris.catalog.Env;
+import org.apache.doris.common.Config;
+import org.apache.doris.proto.InternalService.PGetWalQueueSizeRequest;
+import org.apache.doris.proto.InternalService.PGetWalQueueSizeResponse;
+import org.apache.doris.rpc.BackendServiceProxy;
+import org.apache.doris.system.Backend;
+import org.apache.doris.thrift.TNetworkAddress;
+import org.apache.doris.thrift.TStatusCode;
+import org.apache.doris.transaction.TransactionState;
+import org.apache.doris.transaction.TransactionStatus;
+
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.concurrent.Future;
+
+public class GroupCommitManager {
+
+    public enum SchemaChangeStatus {
+        BLOCK, NORMAL
+    }
+
+    private static final Logger LOG = LogManager.getLogger(GroupCommitManager.class);
+
+    private Map<Long, SchemaChangeStatus> statusMap = new HashMap<>();
+
+    public synchronized boolean isBlock(long tableId) {
+        if (statusMap.containsKey(tableId)) {
+            return statusMap.get(tableId) == SchemaChangeStatus.BLOCK;
+        }
+        return false;
+    }
+
+    public synchronized void setStatus(long tableId, SchemaChangeStatus status) {
+        statusMap.put(tableId, status);
```

Comment:
Logging here can help trace the status changes which can be critical for debugging.
-> LOG.debug("Setting status for tableId {}: {}", tableId, status);